beta-src/src/utils/state/gameApiSlice/reducers/processMapClick.ts

```diff
@@ -89,8 +90,11 @@ export default function processMapClick(state, clickData) {
       }
       state.order.type = phase === "Retreats" ? "disband" : "hold";
     } else if (type === "convoy" && !truthyToTerritory) {
+      const item = data.units[order.unitID];
       state.order.toTerritory = Number(Territory[territoryName]);
-      processConvoy(state);
+      item.type === "Fleet"
```

Comment:
Can't we change this to `data.units[order.unitID]?.type === "Fleet"`? This way we don't need the var above?